language/german/blocks.php

```diff
@@ -3,7 +3,7 @@
 /**
  * Language strings for blocks (de)
  *
- * @package chess
+ * @package    chess
  * @subpackage language
  */
 
```

Comment:
Add phpDoc @ignore to match closing at end of file
```suggestion
/**#@+
 * @ignore
 */
```